usr/bin/cylc

```diff
@@ -177,25 +177,19 @@ if [[ -z "${CYLC_HOME}" ]]; then
     exit 1
 fi
 
-# Note "conda activate" fails to prepend the environment bin dir to PATH if
-# local jobs inherit the scheduler environment and bashrc has prepended other
-# paths here. We don't actually rely on PATH to find "cylc" below, but just in
-# case, this makes "conda activate" do the right thing:
-unset CONDA_SHLVL
-
-# If selecting a virtual environment, activate it
-if [[ -f "${CYLC_HOME}/bin/activate" ]]; then
-    # A Python venv or Conda pack installation
-    . "${CYLC_HOME}/bin/activate" || exit 1
-elif [[ -d "${CYLC_HOME}/conda-meta" && \
-        -f "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh" ]]; then
-    # A normal Conda environment
-    . "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh"
-    conda activate "${CYLC_HOME##*/}" || exit 1
-fi
+# Warn if executable not at $CYLC_HOME
 if [[ ! -x "${CYLC_HOME}/bin/${0##*/}" ]]; then
     echo 1>&2 "ERROR: ${0##*/} not found in ${CYLC_HOME}"
     exit 1
 fi
-# Execute the command in the selected installation.
+# Add CYLC_HOME to the start of the path to ensure the right version of python is used.
+PATH=${CYLC_HOME}/bin:${PATH}
```

Comment:
After this Rose change the PATH manipulation here is subtracted from the Rose app-run environment.

https://github.com/metomi/rose/pull/2506